Task/Tasks/TSKTask.h

```diff
@@ -144,7 +146,7 @@ extern NSString *const TSKTaskDidStartNotification;
  Every TSKTask has an optional delegate that can be informed when a task succeeds or fails. See the
  documentation for TSKTaskDelegate for more information.
  */
-@interface TSKTask : NSObject
+@interface TSKTask<ResultType> : NSObject
```

Comment:
Again, can't comment on the actual line, but at ln 159 I believe `delegate` should be nullable.